Sources/StreamVideo/StreamVideo.swift

```diff
@@ -559,7 +559,7 @@ public class StreamVideo: ObservableObject, @unchecked Sendable {
     ) async throws -> ModelResponse {
         let createDeviceRequest = CreateDeviceRequest(
             id: id,
-            pushProvider: .init(rawValue: pushProvider.rawValue),
+            pushProvider: .init(rawValue: pushProvider.rawValue)!,
```

Comment:
are you sure this should be non-optional? Since previously we could send optional.